### PR TITLE
Fixes Issue # 42 and issue # 43 (previously closed)

### PR DESCRIPTION
**Pull request by RickyDoss**
**The following fixes issue #42:** 
src/assets/svg-logos/youtube-brands.svg
src/components/social_media.js
src/components/social_media_callout.js

### CODE REVIEW

File: src/components/header.js [hunk 1]
```diff
@@ -17,6 +18,9 @@ export default function Header() {
   return (
     <div className="header">
       <Toolbar className={classes.toolbar} disableGutters={true}>
+      <Helmet>
+          <title>SWFL Coders</title> 
+    </Helmet>
```
Comment (digitaldrk) [inline src/components/header.js:23]: Looks like there is an indention issue here

File: src/assets/svg-logos/youtube-brands.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg aria-hidden="true" focusable="false" data-prefix="fab" data-icon="youtube" className="svg-inline--fa fa-youtube fa-w-16" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 576 512"><path fill="currentColor" d="M549.655 124.083c-6.281-23.65-24.787-42.276-48.284-48.597C458.781 64 288 64 288 64S117.22 64 74.629 75.486c-23.497 6.322-42.003 24.947-48.284 48.597-11.412 42.867-11.412 132.305-11.412 132.305s0 89.438 11.412 132.305c6.281 23.65 24.787 41.5 48.284 47.821C117.22 448 288 448 288 448s170.78 0 213.371-11.486c23.497-6.321 42.003-24.171 48.284-47.821 11.412-42.867 11.412-132.305 11.412-132.305s0-89.438-11.412-132.305zm-317.51 213.508V175.185l142.739 81.205-142.739 81.201z"/></svg>
```
Comment (digitaldrk) [inline src/assets/svg-logos/youtube-brands.svg:1]: GIthub seems to want an empty line at the end here:
image

File: gatsby-config.js [hunk 3]
```diff
@@ -32,5 +32,6 @@ module.exports = {
     `gatsby-plugin-sharp`,
     `gatsby-transformer-remark`,
     `gatsby-plugin-sass`,
+    `gatsby-plugin-react-helmet`
```
Comment (digitaldrk) [inline gatsby-config.js:35]: It's weird that this is being added again. It was added in your last PR. I suppose we'll see what happens 🤔 

https://github.com/swfl-coders/website/commit/f25d6f213d67123171927c1e7aeb3573f2ee7f1c#diff-b5e305780d9d473da97c61beab8bc36e5e8871b360942e4686c9b20d8c5d4cfaR35

**Review by digitaldrk — CHANGES_REQUESTED** [review]

File: gatsby-config.js (hunk 3 above)
Comment (RickyDoss) [reply gatsby-config.js:35]: yes i did not know how to get rid of it from this PR after sending it over earlier. Is there any way to remove it from this PR? or are we just going to see if anything happens?

File: src/assets/svg-logos/youtube-brands.svg (hunk 2 above)
Comment (RickyDoss) [reply src/assets/svg-logos/youtube-brands.svg:1]: I checked the other svg files and none have a second blank line under the first with the exception of the coders logo and the github brand svgs, Do you still want me to put a blank line under this one?

**Pull request merged**